### DISCUSSION TITLE
给部分代码添加 typehint 和 详细的 docstring

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#328] [MMDetection]
# 给部分代码添加 typehint 和 详细的 docstring 

给 [https://github.com/open-mmlab/mmdetection/blob/dev-3.x/mmdet/models/detectors/glip.py](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/mmdet/models/detectors/glip.py)  算法增加更详细的 docstring 和 typehint

## Task Information

**Repository**：MMDetection

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：zhanghan0112

**Claim Time**：2023年08月16日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by zhanghan0112** [comment]
Claim the mission
